msteams-platform/messaging-extensions/build-bot-based-message-extension.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Create bot based Message extension
```

Comment:
The Copilot focus on this article feels disparate given its placement in the TOC as a sibling to "Create API-based message extension", which is limited to the core ME (not Copilot) scenario. Is the purpose of this article to cover Copilot ME or to differentiate "traditional"MEs from the new API-based architecture?